spec/requests/users_spec.rb

```diff
@@ -18,16 +18,16 @@
 
   describe 'GET /users/:id' do
     it 'works!' do
-      get user_path(1)
+      get user_path(514)
       expect(response).to have_http_status(200)
     end
   end
   it 'template works!' do
-    get user_path(1)
+    get user_path(514)
     expect(response).to render_template(:show)
   end
   it 'the response body includes correct placeholder text' do
-    get user_path(1)
-    expect(response.body).to include('Here will go the data about the specific user')
+    get user_path(514)
+    expect(response.body).to include('John Doe')
```

Comment:
- [ ] I'm also having errors in this file:

![image](https://github.com/smunoz1988/blog-on-rails/assets/77069011/1d89dced-1808-4629-950f-e1b079fee519)

Please consider using the approach that I just explained in the previous comment to handle the same situation! 🤓 🔧

Reply:
@anagudelogu Fixed!